packages/shared/src/hooks/useActiveNav.tsx

```diff
@@ -16,6 +18,7 @@ export interface UseActiveNav {
 export default function useActiveNav(activeFeed: AllFeedPages): UseActiveNav {
   const router = useRouter();
   const isLaptop = useViewSize(ViewSize.Laptop);
+  const notificationsNavBar = useFeature(feature.notificationsNavBar);
```

Comment:
Here we probably also want to hard enroll only to mobile?
(also note the current one is `!laptop` which might be wrong as we want hard `isMobile`)

Reply:
Added useConditionalFeature and an extra check for !isMobile where it appends the notifications to array.